modules/rest-api/pages/rest-get-memory-information.adoc

```diff
@@ -0,0 +1,100 @@
+= Getting Memory Information
+:description: pass:q[Use REST API to retrieve the current memory quota information for services in a cluster.]
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+[#http-method-and-uri]
+== HTTP Method and URI
+
+----
+GET /pools/default
+----
+
+[#description]
+== Description
+
+Allows to view memory quotas for the Multi-Dimensional Scaling (MDS) services Index, Search, Eventing, and Analytics.
```

Comment:
need to add Data.

Allows to view memory quotas for the Multi-Dimensional Scaling (MDS) services Data, Index, Search, Eventing, and Analytics.